integration-tests/chopsticks/src/tests/xcm/limitedReserveTransferSpiritnetHydraDx.test.ts

```diff
@@ -0,0 +1,120 @@
+import { test } from 'vitest'
+import { sendTransaction, withExpect } from '@acala-network/chopsticks-testing'
+import { u8aToHex } from '@polkadot/util'
+import { decodeAddress } from '@polkadot/util-crypto'
+
+import * as SpiritnetConfig from '../../network/spiritnet.js'
+import * as HydraDxConfig from '../../network/hydraDx.js'
+import { KILT, keysAlice } from '../../utils.js'
+import { spiritnetContext, hydradxContext, getFreeBalanceSpiritnet, getFreeBalanceHydraDxKilt } from '../index.js'
+
+test('Limited Reserve V3 Transfers from Spiritnet Account Bob -> HydraDx', async ({ expect }) => {
+	const { checkEvents, checkSystemEvents } = withExpect(expect)
+
+	// set storage
+	await spiritnetContext.dev.setStorage(SpiritnetConfig.defaultStorage(keysAlice.address))
+	await hydradxContext.dev.setStorage(HydraDxConfig.defaultStorage(keysAlice.address))
+
+	// Create some new blocks to have consistent snapshots
+	await new Promise((r) => setTimeout(r, 50))
+	await spiritnetContext.dev.newBlock()
+	await hydradxContext.dev.newBlock()
+
+	// check initial balance
+	const balanceSovereignAccountHydraDxBeforeTx = await getFreeBalanceSpiritnet(
+		SpiritnetConfig.hydraDxSovereignAccount
+	)
+	expect(balanceSovereignAccountHydraDxBeforeTx).eq(BigInt(0))
```

Comment:
Can you add the Omnipool balance test, to match the post-operation checks?

Reply:
This is already checked. Please have a look on these lines: https://github.com/KILTprotocol/kilt-node/pull/614/files#diff-ce4305e382897732899a9335dad6b6d4350a23b2d2be29fc5c4ca6f864806a48R61-R62